aiida_workgraph/widget/src/widget/__init__.py

```diff
@@ -46,6 +46,14 @@ def from_node(self, node: Any) -> None:
         tdata["label"] = tdata["identifier"]
         for input in tdata["inputs"].values():
             input.pop("property")
+
+        if not show_metadata:
+            inputs = {}
+            for input_k, input_v in tdata["inputs"].items():
+                if not input_k.startswith("metadata"):
```

Comment:
I think we still show the top-level `metadata` port, but not expand it.
```suggestion
                if not input_k.startswith("metadata."):
```